requirements.txt

```diff
@@ -1,6 +1,7 @@
-pytest==2.7.3
+BeautifulSoup==3.2.1
+browserid
```

Comment:
On second thoughts, let's do this in a separate patch as it might not just be a package name change - imports will also need updating.